src/MICore/LaunchOptions.cs

```diff
@@ -411,7 +386,25 @@ public abstract class LaunchOptions
 
         public MIMode DebuggerMIMode { get; set; }
 
+        private Xml.LaunchOptions.BaseLaunchOptions _baseOptions;
```

Comment:
Does base options remove any option that doesn't make sense like processid? I assume that's the motivation behind the refactoring.